### PR TITLE
Istio Migration Guide

### PR DESCRIPTION
**Pull request by rosemarymarano**
First pass at the Istio to OCNE 2.0 Migration Guide

### CODE REVIEW

File: content/en/docs/guides/migrate/istio.md [hunk 1]
```diff
@@ -0,0 +1,1746 @@
...
+          seccompProfile:
+            type: RuntimeDefault
+        service:
+          externalIPs:
+          - bb.bb.bb.bb1
+          - bb.bb.bb.bb2
+          - bb.bb.bb.bb3
+          - bb.bb.bb.bb4
+          ports:
+          - name: status-port
```
Comment (pfmackin) [inline content/en/docs/guides/migrate/istio.md:124]: This is not the default configuration and it is was too big.  I will get the configuration that we use.

File: content/en/docs/guides/migrate/istio.md [hunk 2]
```diff
@@ -0,0 +1,1746 @@
...
+</div>
+{{< /clipboard >}}
+
+<details>
+<summary><b>Example istio-profile.yaml</b></summary>
+
+{{< clipboard >}}
+<div class="highlight">
+
+```
```
Comment (pfmackin) [inline content/en/docs/guides/migrate/istio.md:861]: Similare comment as above, replace this will the YAML I gave you.

Comment (rosemarymarano) [reply content/en/docs/guides/migrate/istio.md:332]: @pfmackin I used the YAML you gave me to replace the Example iop.yaml file that is located before this example. Are you saying that you want me to use that same YAML file to replace this Example istio-profile.yaml file?

**Comment by rosemarymarano** [comment]
Content was used elsewhere.

**Pull request closed without merging**